syft/frameworks/crypten/__init__.py

```diff
@@ -39,18 +42,29 @@ def get_worker_from_rank(rank: int, cid: int = None) -> BaseWorker:
 
 
 def load(tag: str, src: int, **kwargs):
+    if tag.startswith("crypten_model"):
+        worker = get_worker_from_rank(src)
+        results = worker.search(tag)
+        assert len(results) == 1
+
+        result = results[0]
+
+        if isinstance(result, ObjectPointer):
+            model = result.clone().get()
+        else:
+            model = result
+
+        return utils.onnx_to_crypten(model.serialized_model)
+
```

Comment:
Maybe it's worth having a different function for loading models? It will give more flexibility on the tag name for the user, instead of telling him that he have to prefix the tag with "crypten_model"...

Reply:
Removed this. From our Slack discussion, this involves using the Syft internals when running the CrypTen computation and we do not want this. The serialization of the model is done in the same way as with Jails.